<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BuksanSpy</class>
 <widget class="QMainWindow" name="BuksanSpy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>898</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BuksanSpy</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QPushButton" name="pushButton_2">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Включить запись 
 со всех камер</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="6">
     <layout class="QGridLayout" name="CamLayout">
      <item row="0" column="0">
       <widget class="QTabWidget" name="Cams_TabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="AllCams">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <attribute name="title">
          <string>Все</string>
         </attribute>
         <widget class="QWidget" name="gridLayoutWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>651</width>
            <height>401</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="camslay"/>
         </widget>
        </widget>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Page</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButton_4">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Включить запись 
 с выбранной камеры</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="pushButton_3">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Журнал</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>898</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="separator"/>
    <addaction name="InitCams_action"/>
    <addaction name="StartAllCams_Action"/>
    <addaction name="StartCurrentCam_Action"/>
    <addaction name="Quit_Action"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Видеозаписи</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Администрирование</string>
    </property>
    <widget class="QMenu" name="menu_4">
     <property name="title">
      <string>Пользователи и группы</string>
     </property>
     <addaction name="users_Action"/>
     <addaction name="MandatoryGroups_Action"/>
    </widget>
    <widget class="QMenu" name="menu_5">
     <property name="title">
      <string>Устройства</string>
     </property>
     <addaction name="ShowDevList_Action"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="menu_4"/>
    <addaction name="separator"/>
    <addaction name="menu_5"/>
    <addaction name="eventlog_Action"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <action name="StartAllCams_Action">
   <property name="text">
    <string>Включить запись со всех камер</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+A</string>
   </property>
  </action>
  <action name="StartCurrentCam_Action">
   <property name="text">
    <string>Включить запись с выбранной камеры</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
  </action>
  <action name="Quit_Action">
   <property name="text">
    <string>Выйти из программы</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Q</string>
   </property>
  </action>
  <action name="users_Action">
   <property name="text">
    <string>Пользователи</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+U</string>
   </property>
  </action>
  <action name="MandatoryGroups_Action">
   <property name="text">
    <string>Мандатные группы</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+M</string>
   </property>
  </action>
  <action name="AddDevice_Action">
   <property name="text">
    <string>Добавить устройство</string>
   </property>
  </action>
  <action name="InitCams_action">
   <property name="text">
    <string>Инициализировать устройства</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+I</string>
   </property>
  </action>
  <action name="AddDev_Action">
   <property name="text">
    <string>Добавить устройство</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+N</string>
   </property>
  </action>
  <action name="ShowDevList_Action">
   <property name="text">
    <string>Показать список устройств</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="eventlog_Action">
   <property name="text">
    <string>Журнал событий</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+L</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
